distributed/worker.py

```diff
@@ -647,6 +648,7 @@ def __init__(
             ("ready", "released"): self.transition_generic_released,
             ("released", "error"): self.transition_generic_error,
             ("released", "fetch"): self.transition_released_fetch,
+            ("released", "missing"): self.transition_released_fetch,
```

Comment:
Typo - this should be ``transition_generic_missing``.
This causes an infinite transition loop (https://github.com/dask/distributed/issues/6305):
1. The scheduler calls ``handle_compute_task`` with an empty who_has. This is a broken use case to begin with, but it is happening and the worker should either cope with it gracefully or crash loudly in ``handle_compute_task`` itself.
2. ``handle_compute_task`` creates the dependencies with ``ensure_task_exists`` in state=released and requests a  transition to fetch
3. The released->fetch transition, handled by ``transition_released_fetch``, notices that who_has is empty, so instead of transitioning to fetch it returns ``{ts: missing}`` and keeps the task in released state
4. The released->missing transition, erroneously handled by the same method, repeats point 3 forever.

Reply:
Is this "fixed" by https://github.com/dask/distributed/pull/6318?

Reply:
Yes.